core/test/functional/api/v0.1/webhooks_spec.js

```diff
@@ -81,7 +81,8 @@ describe('Webhooks API', function () {
                         var jsonResponse = res.body;
 
                         should.exist(jsonResponse.webhooks);
-                        testUtils.API.checkResponse(jsonResponse.webhooks[0], 'webhook');
+                        testUtils.API.checkResponse(jsonResponse.webhooks[0], 'webhook',
+                            null, ['name', 'integration_id', 'secret', 'last_triggered_at', 'api_version']);
```

Comment:
I'm a bit opposed for the `last_` usage here :smile: It's super explicit this way which is nice, but: `updated_at` has the same meaning - last update date and is not named `last_updated_at`, because that's what we mean when storing the date there anyways. Not super critical just my 2 :money_with_wings:

Reply:
That's a good point, and good example, because I liked the explicit part about it as well! 😄 We can move to `triggered_at`, but I guess its not super urgent so will def revisit this once more before the final migrations happen for schema 😸

Reply:
We do have some other `last_` columns too.

`user.last_seen`
`api_key.last_seen_at`
`api_key.last_seen_version`

Reply:
The `api_key` example is most relevant to this discussion I think because it shows what happens when you have non-datetime values - `api_key.seen_version` doesn't make as much sense 😄 

If we're adding `webhook.last_triggered_error` or similar it also hints that the more explicit `last_` convention is helpful 🤔